gcs/deleteFile/proto/deleteFile.proto

```diff
@@ -39,10 +37,6 @@ message DeleteFileProbe {
   // The name of the bucket that contains the object to be deleted.
   optional string bucketName = 2;  
 
-  // REQUIRED
-  // The name of the object to be deleted.
-  optional string objectName = 3;
```

Comment:
It's OK to remove a field that was never used.
When doing removal where it's not guaranteed that the field was never used, mark the removed field name and its tag as reserved to avoid accidentally re-using them:

```
reserved 3;
reserved "objectName";
```

Reply:
Okay 😄 Thanks for explaining how to properly remove a proto field when in use!